lib/brewdler/brew_installer.rb

```diff
@@ -2,14 +2,14 @@ module Brewdler
   class BrewInstaller
     def self.install(name, options = {})
       if `which brew`; $?.success?
```

Comment:
Updated. Now it's using `brew --prefix` and `brew cask --version` to check `brew` and `cask`.